webui/react/src/pages/Settings/SettingsAccount.test.tsx

```diff
@@ -49,4 +68,18 @@ describe('SettingsAccount', () => {
     const editor = container.querySelector(`[data-value="${DISPLAY_NAME}"]`);
     expect(editor).toBeInTheDocument();
   });
+  it('should be able to change display name', async () => {
+    setup();
+    await user.type(screen.getByPlaceholderText('Add display name'), 'a');
+    await user.keyboard('{enter}');
+    expect(mockPatchUser).toHaveBeenCalledWith({
+      userId: 1,
+      userParams: { displayName: `${DISPLAY_NAME}a` },
+    });
+  });
+  it('should be able to view change password modal when click', async () => {
```

Comment:
teeny nit:change test name to just **should be able to view change password modal**, the `when click` is unclear to me since I could not tell what is being clicked.